Test/git-issues/git-issue-1963a.dfy.expect

```diff
@@ -1,2 +1,2 @@
-git-issue-1963a.dfy(12,15): Error: Refinement cannot change the base type of a `newtype`.  To refine `NT`, either omit this `...` or omit the `=` sign and the parent type's body.
+git-issue-1963a.dfy(12,15): Error: Refinement cannot change the base type of a newtype. To refine 'NT', either omit this '...' or omit the '=' sign and the parent type's body.
```

Comment:
I was not able to resolve the error by following this message. I only ran into further errors.

Reply:
Following the message I tried (separately and combined):
1) omit `...`
2) omit `= x: int | x >= 0 witness 0`